subsys/bluetooth/host/gatt.c

```diff
@@ -1406,9 +1478,13 @@ int bt_gatt_notify_cb(struct bt_conn *conn, const struct bt_gatt_attr *attr,
 		      bt_gatt_complete_func_t func)
 {
 	struct notify_data nfy;
+	u16_t handle;
 
-	__ASSERT(attr && attr->handle,
-		 "invalid parameters\n");
+	__ASSERT(attr, "invalid parameters\n");
+
+	handle = attr->handle ? : find_static_attr(attr);
+
+	__ASSERT(handle, "invalid parameters\n");
```

Comment:
Also an issue in the old code, but especially converting this to call `find_static_addr()` makes the assert look strange. Wouldn't `return -ENOENT` make more sense?

Reply:
Perhaps, it was asserting because that indicates something is not right with the attribute if it had its handle set to 0, but perhaps switching to -ENOENT so it becomes a runtime error makes more sense with find_static_attr.